docs/zh/packages/r/CallbackFunctionTutorial.md

```diff
@@ -0,0 +1,238 @@
+# MXNet R教程之回调函数
+
+本文将概述我们如何在模型训练的时候使用或者自定义一些回调函数。本教程使用 Rmarkdown 编写。
+
+- 你可以直接看我们在主站上的教程： [MXNet R Document](http://mxnet.readthedocs.org/en/latest/R-package/CallbackFunctionTutorial.html)
```

Comment:
the correct url is http://mxnet.readthedocs.io/en/latest/packages/r/CallbackFunctionTutorial.html